schema.graphql

```diff
@@ -58,4 +62,17 @@ type Trade @entity {
   buyToken: Token!
   "Order"
   order: Order!
+}
+
+type Settlement @entity {
+  "TxHash"
+  id: ID!
+  "Transaction hash"
+  txHash: Bytes!
+  "First trade timestamp"
+  firstTradeTimestamp: BigInt!
+  "Collection of trades"
+  trades: [Trade!] @derivedFrom(field: "settlement")
+  "Sum of the fees in trades"
+  totalFees: BigInt!
```

Comment:
What currency is this fee in?

I think it would only make sense to have this field when we are able to convert from arbitrary tokens into USD and/or ETH

That is because the fee for each trade is taken always in the sell token.

Reply:
Makes sense, mixed currencies. Already fixed!